<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoonWeather</class>
 <widget class="QWidget" name="MoonWeather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto Slab</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MoonWeather</string>
  </property>
  <widget class="QLabel" name="hour_lbl">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>30</y>
     <width>1021</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>130</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>88:88:88</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="date_lbl">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>190</y>
     <width>1021</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VENDREDI 24 SEPTEMBRE 2021</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="weather_pic">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>256</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="city_lbl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CITY</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="temperature_lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>70</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100Â°</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="conditions_lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PARTIELLEMENT NUAGEUX</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="wind_pic_lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pressure_pic_lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>460</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="humidity_pic_lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="wind_speed_lbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100 Km/h</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="humidity_percent_lbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pressure_rate_lbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>460</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>8888 hPa</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="longitude_lbl">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LONGITUDE : x.x</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="latitude_lbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LATITUDE : xx.xxxxx</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="visibility_pic_lbl">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="visibility_km_lbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>530</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSans</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100 Km</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
